components/service/glean/README.md

```diff
@@ -71,11 +71,16 @@ excellent place to perform this operation is within the `onCreate` method of the
 Android's `Application` class.
 
 ```Kotlin
+import org.mozilla.yourApplication.GleanMetrics.Pings
+
 class SampleApplication : Application() {
 
     override fun onCreate() {
         super.onCreate()
 
+        // OPTIONAL: Initialize your application's custom pings, if it has any.
```

Comment:
nit: The wording of this comment seems a little unclear to me.  Maybe this could be worded more like:
```
// If you have custom pings in your application, you must register them using the following command. 
// This command should be omitted for applications not using custom pings.
```

Reply:
Good point.  Thanks.